packages/components/src/ui/text/stories/index.js

```diff
@@ -58,44 +51,3 @@ export const highlight = () => {
 		</Text>
 	);
 };
-
-const Adapted = ( { doAdapt, ...props } ) =>
```

Comment:
I removed this story since it's covered in the original `Text` component as well.

Reply:
It's not really covered by the original text component because it's not possible to run Storybook with the "next" flag for G2 components (at least not as far as @ItsJonQ and I could tell). So I added this as the best way to compare the current component with the G2 one wrapped in the adapter.

Reply:
I see. We can remove the flag this week after WP 5.7 beta 1 is ready and adjust stories if necessary 😃

Reply:
Sounds good.